OracleAccessManagement/dockerfiles/12.2.1.4.0/container-scripts/createDomainAndStart.sh

```diff
@@ -0,0 +1,275 @@
+#!/bin/bash
+#
+#
+#
+#
+# Copyright (c) 2019-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
It should be 2019, 2020. We also no longer include the phrase "All rights reserved."

Reply:
taken care of this.